### PR TITLE
feat: #44 improve error display

### PR DESCRIPTION
**Pull request by dyon21**
[#44 Do not override the message with error.](https://github.com/ZenGPT/confluence-gpt/issues/44)

Errors now be displayed sepetately from messages:

### CODE REVIEW

File: views/components/MarkdownRenderer.jsx [hunk 1]
```diff
@@ -168,18 +170,21 @@ const MarkdownRenderer = ({ content, loading, isGPT }) => {
...
+              chatMode
+              remarkPlugins={[remarkGfm]}
+              components={{
+                code: CodeBlock,
+              }}
+            >
+              {content}
+            </StyledMarkdown>
+          )}
+          <ErrorMessage>{error}</ErrorMessage>
```
Comment (MrCoder) [inline views/components/MarkdownRenderer.jsx:185]: Do we need to make it conditional? Like `{ error && (<ErrorMessage...`?

Comment (dyon21) [reply views/components/MarkdownRenderer.jsx:185]: Sure, I had added it at the beginning, but it seems I accidentally removed it after making some changes

File: views/hello-world.jsx [hunk 2]
```diff
@@ -109,14 +85,15 @@ const FormDefaultExample = () => {
...
+        //   refresh quota
       } catch (e) {
+        console.error(e);
         setSessions((prev) => {
           const updatedSessions = prev.map((chat) => {
             if (chat.id === chatId) {
               return {
                 ...chat,
-                message: 'An error occurred, please try again later.',
+                error: 'An error occurred, please try again later.',
```
Comment (MrCoder) [inline views/hello-world.jsx:96]: An error occurred. Please try again laster, or submit a ticket at https://zenuml.atlassian.net/servicedesk/customer/portals.

**Review by MrCoder — CHANGES_REQUESTED** [review]

**Comment by whimet** [comment]
Close this one as it's too old.

**Pull request closed without merging**